contrib/kubespray/boot.sh

```diff
@@ -56,6 +56,8 @@ ansible-playbook -i ${HOME}/pai-deploy/cluster-cfg/hosts.yml drivers-install.yml
 
 ansible-playbook -i ${HOME}/pai-deploy/cluster-cfg/hosts.yml docker-runtime-setup.yml || exit $?
 
+ansible-playbook -i ${HOME}/pai-deploy/cluster-cfg/hosts.yml set-host-daemon-port-range.yml || exit $?
```

Comment:
do you make sure the `net.ipv4.ip_local_port_range` is only configured on worker nodes?

Reply:
Sure, only run on worker node